src/app/[locale]/viewer/_components/submodel/SubmodelDetailComponentProps.ts

```diff
@@ -0,0 +1,6 @@
+import { Submodel } from '@aas-core-works/aas-core3.0-typescript/types';
+
+
+export type SubmodelDetailComponentProps = {
```

Comment:
Currently unused, could be used in the GenericSubmodelComponent. We could also merge this structure with the CustomTreeItemContentProps, ContactInformationComponentProps and AddressComponentProps by adding one or two optional values